apps/central/src/components/SchemaCreateModal.vue

```diff
@@ -98,6 +119,11 @@ export default {
     endpoint() {
       return "/api/graphql";
     },
+    host() {
+      if (window) {
+        return window.location.origin;
+      }
```

Comment:
else ?

Reply:
this was added in light of server side rendering. Else it will not work :-) Do you have recommendation?

Reply:
in case of ssr host will return null, but i need to look in to the use of the host var here. For now i do not see a problem as the central add does not run via ssr